src/deep-cache/lib/SharedCache.js

```diff
@@ -70,11 +71,42 @@ export class SharedCache {
    * @returns {String}
    */
   buildKeyFromLambdaRuntime(runtime) {
-    let payload = JSON.stringify(runtime.request.data);
+    return runtime.context && runtime.context.has('invokedFunctionArn') ?
+      `${runtime.context.getOption('invokedFunctionArn')}#${SharedCache._stringifyPayload(runtime.request.data)}` :
+      null;
+  }
+
+  /**
+   * @param {Object} payload
+   * @private
+   */
+  static _stringifyPayload(payload) {
+    return JSON.stringify(
+      SharedCache._normalizeObject(payload)
+    );
+  }
+
+  /**
+   * @param {Object} data
+   * @returns {Object}
+   * @private
+   */
+  static _normalizeObject(data) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).](https://www.codacy.com/app/MitocGroup/deep-framework/file/1784205094/issues/source?bid=2895475&fileBranchId=3068904#l95)